include/verilatedos.h

```diff
@@ -604,7 +604,7 @@ reverse_wrapper<T> reverse_view(const T& v) {
 
 // C++17's std::as_const
 template <class T>
-T const& as_const(T& v) {
+T const& as_const(T& v) VL_MT_SAFE {
```

Comment:
I'm not sure of this, if it is accessing something unsafe casting to const doesn't make it safe.  Maybe the argument is that doesn't matter as a later call will still get checked correctly.  If so, add a comment on this please.

Reply:
Yes, this ``VL_MT_SAFE`` only applies to this function. Any call that is made on object returned by this function still needs to be annotated properly.